.github/workflows/CI.yml

```diff
@@ -12,25 +12,26 @@ concurrency:
   cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
-    runs-on: ${{ matrix.os }}
+    name: Julia ${{ matrix.version }} - ${{ matrix.platform.os }} - ${{ matrix.platform.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.platform.os }}
     strategy:
       fail-fast: false
       matrix:
+        platform:
+          - os: ubuntu-latest
+            arch: x64
+          - os: ubuntu-24.04-arm
```

Comment:
Shouldn't we use `ubuntu-latest` here as well? It is odd to have an inconsistency between x86 and ARM.


More broadly, I think `ubuntu-latest` is sufficient here. Julia has far fewer distro specific issues than something that might have to compile with the native c++ compiler.

Reply:
No.  `ubuntu-latest` is x86-64, `ubuntu-24.04-arm` is the aarch64 image, there's no "latest" alias, see https://github.com/actions/partner-runner-images/

Reply:
OK - I didn't realise that, thanks! Looking at https://github.com/marketplace/actions/run-on-architecture#supported-platforms, is it then `ubuntu_latest` for ARM, to achieve the same result? My mild concern is that we end up leaving the ARM distro behind the x86 one.

Reply:
I think that's using qemu, which you really don't want to use if you can possibly avoid it.

Reply:
OK, we go with this PR as written then!